src/components/Todofilter.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import classNames from 'classnames';
+import { FilterType } from '../types/FilterType';
+
+type Props = {
+  filterType: FilterType;
+  onFilterChange: (filter: FilterType) => void;
+  todosLeftActive: number,
```

Comment:
In my opinion not very good naming, it's better to use ''remainingTodos'' or something more understandable.